dll/win32/shell32/debughlp.cpp

```diff
@@ -379,6 +379,7 @@ BOOL pcheck( LPCITEMIDLIST pidl )
                 case PT_YAGUID:
                 case PT_IESPECIAL2:
                 case PT_SHARE:
+                case 0x99:      /* Network Connection pidl type */
```

Comment:
Make it a `#define`?

Reply:
Netshell itself does not even have a define for this, so this is following the existing style.